packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -1243,6 +1226,51 @@ module('Search', setupModule, () => {
     });
 });
 
+module('Date AM/PM Handling', {
+    beforeEach: function() {
+        setupModule.beforeEach.call(this);
+        this.instance.option('value', null);
+        ['a', 'aa', 'aaa', 'aaaa', 'aaaaa'].forEach((format) => {
```

Comment:
`beforeEach` is called before each test
=> before each test u iterate through all display formats setting it one after another and finally u always have the last `aaaaa` format in all your tests.

We cannot set `displayFormat` in beforeEach.